models/ConfigureForm.php

```diff
@@ -89,6 +93,7 @@ public function rules()
         return [
             [['enableEmailGoService', 'disableAuthChoicesIos'], 'boolean'],
             [['senderId'], 'number'],
+            [['firebaseProjectId', 'firebaseApiKey', 'firebaseAppId', 'firebaseVapidKey'], 'string'],
```

Comment:
Should be set to required.

Reply:
@ArchBlood The fields "Sender ID" and "Service Account (JSON file)" are not required.
They were used to check the dirver `humhub\modules\fcmPush\driver\Fcm` is configured in previous version.
Now to check the driver is configured we check the 2 old fields and 4 new fields, so I am not sure we should make the 4 new fields required.